ci/build-and-test-nupic-bindings.sh

```diff
@@ -88,12 +94,11 @@ echo "RUNNING NUPIC BINDINGS BUILD: BUILD_TYPE=${BUILD_TYPE}, " \
 # Install pycapnp to get the matching capnproto headers for nupic.core build
 # NOTE Conditional pycapnp dependency should be incorporated into
 # bindings/py/requirements.txt to abstract it from upstream scripts.
-pip install pycapnp==0.5.8
+pip install ${PIP_USER} pycapnp==0.5.8
 
 # Install nupic.bindings dependencies; the nupic.core cmake build depends on
 # some of them (e.g., numpy).
-pip install -r ${NUPIC_CORE_ROOT}/bindings/py/requirements.txt
-
+pip install --ignore-installed ${PIP_USER} -r ${NUPIC_CORE_ROOT}/bindings/py/requirements.txt
```

Comment:
I had to add `--ignore-installed` here or the numpy installation would fail with a `ReadTimeout` error when installing from PyPi. I have no clue why, but more details on https://jira.numenta.com/browse/DEVOPS-181.